### DISCUSSION TITLE
Difference between `single` and `find`?

### DISCUSSION THREAD

**Discussion by Sawtaytoes** [ReactiveX/rxjs#6107] [Q&A] [answered]
I've been going through the docs site trying to understand every operator I don't already know.

In one of those instances, I noticed `single` sounds an awful lot like `find`. What's the difference?

**Comment by madebyjeffrey** [comment, marked as answer]
Read the documentation carefully, you will notice that single will throw an error in the event more than one item emitted, while find emits no errors. There are cases where you only want one item and other cases where you must only have one item.

In C# LINQ there is also a single and first that has the same kind of difference.
